__tests__/getNearbyPlaces.test.ts

```diff
@@ -30,11 +30,15 @@ const mockPlace: GooglePlace = {
 
 const mockPlaceA11yData: Place = {
 	_id: mockPlace.place_id,
-	avgBraille: 5,
-	avgFontReadability: 5,
-	avgGuideDogFriendly: 5,
-	avgNavigability: 5,
-	avgStaffHelpfulness: 5,
+	guideDogAvg: 5,
+	isMenuAccessibleAvg: 5,
```

Comment:
```suggestion
	isMenuAccessibleAvg: 1,
```
all of the `is` fields only should accept 0 or 1.

Reply:
whoops!! Thanks for catching that!